### ISSUE TITLE
Line to be removed in heights-post.stan within document

### ISSUE THREAD

**Issue by magland** [bob-carpenter/stan-getting-started#1]
https://github.com/bob-carpenter/stan-getting-started/blob/9e58217530cc714c47cab61f0beb07f6802c6204/quarto/stan-getting-started.qmd#L3074

This line needs to be removed -- it is correct in the .stan file, but not in the actual document.

**Comment by bob-carpenter** [comment]
Thanks.  Fixed.  Does anyone know a good way to include code in a markdown document such that it gets syntax highlighted?

**Comment by rfhb** [comment]
> Does anyone know a good way to include code in a markdown document such that it gets syntax highlighted?

@bob-carpenter Not sure if this is what you were looking for, but what about `#| eval: false` in the first line of a code chunk, see https://quarto.org/docs/reference/cells/cells-jupyter.html#code-output?

**Comment by magland** [comment]
> Does anyone know a good way to include code in a markdown document such that it gets syntax highlighted?

Aren't you already doing that with three backtics and the language name?

**Comment by bob-carpenter** [comment]
@rfhb & @magland:  The Stan programs live in their own files.  I'd like to include them into the markdown document, but also retain the syntax highlighting.  I can include without syntax highlighting using Python or I can syntax highlight by cutting-and-pasting.

**Comment by rfhb** [comment]
@bob-carpenter This extension works well for me to include external source file contents with syntax highlighting in the rendered Quarto document: https://github.com/quarto-ext/include-code-files, one of the many useful Quarto extensions at https://quarto.org/docs/extensions/ Not sure if you additionally were looking for having the file content displayed in the unrendered qmd document, perhaps this could be asked of the developers of extensions such as https://marketplace.visualstudio.com/items?itemName=quarto.quarto.

**Comment by bob-carpenter** [comment]
*[Edit:  D'oh!  This was user error on my part.  I was using the wrong magic name (I'm very bad at name matching in configs, which is why I dislike using high-level frameworks for things).  Even when I fixed that it didn't work, though :-(. ]*

*[Edit 2:  Path problems.  Now they're fixed.  I was expecting a general install, but I had to move the downloaded file parallel to my .qmd docs]*

Thanks for the suggestion, @rfhb.  It'd be nice if it worked.  I tried it and then filed an issue about how it didn't work:

https://github.com/quarto-ext/include-code-files/issues/6

Looking at other issues, this doesn't look to be a well-engineered or well-maintained extension, and I'd really like other people to be able to run this, so I'm not going to add anything that's an installation headache.

**[Edit:  My bad.  The system wasn't broken, it just wasn't documented clearly enough for me (I'm very bad at IT).  I submitted a PR with the fix that what worked for me.]**
